wellcomeml/ml/keras_utils.py

```diff
@@ -0,0 +1,17 @@
+from sklearn.metrics import f1_score, precision_score, recall_score
+import tensorflow as tf
+
+class Metrics(tf.keras.callbacks.Callback):
```

Comment:
i like this

Reply:
That's very useful - I'd like to add that to the semantic similarity as well. I think it might be possible.